locations/spiders/mauboussin.py

```
@@ -0,0 +1,34 @@
+
+class MauboussinSpider(CrawlSpider, StructuredDataSpider):
+    name = "mauboussin"
+    item_attributes = {
+        "brand": "Mauboussin",
+        "brand_wikidata": "Q3300085",
+    }
+    allowed_domains = ["boutiques.mauboussin.fr"]
+    start_urls = [ "https://boutiques.mauboussin.fr/boutiquesmauboussin/en/load-stores/location_country/whosonfirst:country:85633147"]
+    drop_attributes = {"image", "twitter", "facebook"}
+
+    #restrict_xpaths used to keep only mauboussin stores, not other stores selling mauboussin products
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)